CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## 43 Release Notes
 
+### Call caching database refactoring
+
+Cromwell's `CALL_CACHING_HASH_ENTRY` primary key has been refactored to use a `BIGINT` datatype in place of the previous
+`INT` datatype. Cromwell will not be usable during the time the Liquibase migration for this refactor is running.
+In the Google Cloud SQL environment this migration runs at a rate of approximately 100,000 `CALL_CACHING_HASH_ENTRY`
+rows per second. In deployments with millions or billions of `CALL_CACHING_HASH_ENTRY` rows the migration may require  
+a significant amount of downtime so please plan accordingly.
```

Comment:
Any advice on the best way to calculate the table size? My naive `select count(*) from ...` often turns out to not be a very good idea

Reply:
I use
```
SELECT * FROM information_schema.innodb_sys_tablestats;
```
which is instant

Reply:
looks like one needs to be root for that though